lib/ts/recipe/thirdpartyemailpassword/recipe.tsx

```diff
@@ -65,6 +66,12 @@ export default class ThirdPartyEmailPassword extends AuthRecipe<
             typeof ThirdPartyEmailPasswordWebJS
         > = ThirdPartyEmailPasswordWebJS
     ) {
+        if (
+            SuperTokens.usesDynamicLoginMethods === false &&
+            config.thirdPartyConfig?.signInAndUpFeature.providers.length === 0
+        ) {
+            throw new Error("ThirdParty signInAndUpFeature providers array cannot be empty.");
```

Comment:
This can be empty (which means the thirdparty section is disabled) if email password is not disabled.

Reply:
do you want me to remove this check ?

Reply:
No. This only needs to throw if all of the following is true:
1. email password is disabled (this would be passwordless in the thirdpartypasswordless case)
2. thirdparty provider list is empty
3. not using dynamic login methods.